scrapi/base/__init__.py

```diff
@@ -38,6 +58,14 @@ def normalize(self, raw_doc):
     def lint(self):
         return lint(self.harvest, self.normalize)
 
+    @property
```

Comment:
Possibly? We should certainly space them out but that could be auto incremented by the registry.